account_move_report/report/account_move_report.py

```diff
@@ -0,0 +1,27 @@
+from odoo import api, models
+
+
+class ParticularReport(models.AbstractModel):
+    _name = 'report.account_move_report.account_entries_report'
+
+    @api.multi
+    def get_report_values(self, docids, data=None):
+        docs = self.env['account.move'].browse(docids)
+        total_debit_credit = self.get_total_debit_credit(docs)
+        return {
+            'doc_ids': docs.ids,
+            'doc_model': 'account.move',
+            'data': data,
+            'docs': docs,
+            'total_debit_credit': total_debit_credit,
+        }
+
+    def get_total_debit_credit(self, docs):
+        res = {}
```

Comment:
It can be faster using:
 - `read_group()`
For a new PR
Similar to https://github.com/odoo/odoo/pull/31313